model/code/registry.yaml

```diff
@@ -9,15 +9,18 @@ groups:
         type: string
         stability: development
         brief: >
-          The method or function name, or equivalent (usually rightmost part of the code unit's name).
-        examples: serveRequest
-      - id: code.namespace
-        type: string
-        stability: development
-        brief: >
-          The "namespace" within which `code.function.name` is defined. Usually the qualified class or module name,
-          such that `code.namespace` + some separator + `code.function.name` form a unique identifier for the code unit.
-        examples: com.example.MyHttpService
+          The method or function fully-qualified name without arguments. The value should fit the natural representation of the language
+          runtime, which is also likely the same used within `code.stacktrace` attribute value.
+        examples: >
```

Comment:
I think examples is supposed to be an array, maybe include just a couple of short-ish examples, and move the complete set of examples to the notes section?

Reply:
Done in 467ce34